---
title: Structural Bioinformatics
keywords: optional
search: exclude
contributors: @bordin89, @sillitoe, @gtauriello
tags: [<!---REPLACE THIS with comma separated list of (outgoing) tags to other pages related to this page--->]
---

## Introduction

Structural bioinformatics provides the scientific methods and tools to analyse, archive, validate, and present the biomolecular structure data generated by the structural biology community. It also provides an important link with the genomics community, as structural bioinformaticians also use the extensive sequence data to predict protein structures, their complexes and their functional sites. A number of key activities generating data outputs that can be or are being captured in public repositories include:

-	Prediction of protein structures
-	Prediction of protein functional sites
-	Prediction of protein complexes
-	Prediction of protein ligand interactions
-	Prediction of protein nucleic acid interactions
-	Prediction of the impacts of residue mutations on protein binding energies

Most researchers use publicly available data from large public repositories (e.g. protein structural data from the PDB (http://www.wwpdb.org/), PDBe (https://www.ebi.ac.uk/pdbe/) or NDB (http://ndbserver.rutgers.edu/), protein sequence data from UniProt (https://www.uniprot.org/), NCBI (https://www.ncbi.nlm.nih.gov/genbank/), ligand data from ChEBI (https://www.ebi.ac.uk/chebi/), ChEMBL (https://www.ebi.ac.uk/chembl/),  as input for their methods. However, a wide variety of protein structure, protein sequence, chemical ligand, nucleic acid features are used in the predictions and various machine learning strategies applied. Different training approaches are used. Some methods require specific subsets of the data e.g. more evolutionary distant, more information rich sequence subsets. All these factors can affect the predictions and so could be considered necessary metadata for the predictions generated.

Another challenge is that groups use different approaches for assessing the confidence in their predictions and predicting the likely accuracy.    

Whilst individual groups may make their prediction data or prediction servers publicly available, currently very few repositories exist to integrate these prediction data on a large scale.

The 3D-BioInfo Community is currently working to establish common ontologies and standard benchmark datasets for these activities in order to integrate the data and compare methodologies more reliably. Please contact the coordinators of these activities (https://elixir-europe.org/communities/3d-bioinfo) if you have queries or would like to participate in the working groups drawing up these standards.

To date, most progress has been made in the areas of
1.	Deposition of theoretical models of macromolecular structures in the ModelArchive (https://www.modelarchive.org). This repository complements the PDB archive for experimental structures and PDB-Dev for integrative structures.
2.	Integration of protein structure data via Genome3D (http://www.genome3d.net/). This work is now being taken up and extended by the 3D-Beacons network (https://github.com/3D-Beacons) and PDBe-Knowledgebase which will archive the 3D-models and make them available.
3.	Assessment of accuracy for protein structure predictions by CAMEO (https://www.cameo3d.org/) and CASP (https://predictioncenter.org/)
4.	Integration of predicted functional sites and predicted mutation impacts for proteins (https://www.ebi.ac.uk/pdbe/pdbe-kb) 

## Structure Prediction

## Introduction
 
Protein structure prediction methods depend on the similarity of the target sequence to known protein structures. For close relatives (sequence identity > 30% identity of E-value 10-3) homology modelling can be used. For more distant relatives, threading or ab-initio approaches are often necessary. The latter often rely on multiple sequence alignments of homologues typically identified using searches of large databases e.g. UniProt, using powerful methods e.g. HHblits.

The quality of the resulting model will determine the applications for which it can be used e.g. high quality models are needed for drug design. Hence, it is critical for models to include accurate estimates of its predicted quality. 

### Description

FAIR principles require that relevant information on input query protein sequence data are captured, together with information on any protein structures used in the generation of the predicted structure. The strategies used to identify any protein homologues used in the prediction (e.g. in a MSA) should also be captured, including the protein sequence repository, the search strategy, any thresholds used in recognising homologues and the alignment method).

There should be links to information on the methodology used to predict the accuracy of the model and to assign a confidence level in that prediction.   

### Considerations <!--- (optional) --->

- Do you have an identifier for the protein sequence you modelled (e.g. UniProtID or other sequence repository such as MGnify)?
- Is their experimental structure available for this protein? The whole structure or just part of it? 
- Do you have an identifier for any structural templates you used to build the structure (e.g. PDB identifier)?
- Can you describe the strategies you used for extracting and aligning any homologues used by the method? 
- Can you describe the key methodologies used in this prediction?
- Do you have a method for assigning confidence and model quality (gen.idea of structure -> drug design)?


### Solutions

- Protein sequence identifiers are provided by a number of public repositories (see below)
- Protein structure identifiers are provided by the wwPDB (see below)
- To determine whether any experimental structural data exists for this protein you can search the Protein Data Bank (PDB) with the target sequence in FASTA format
- Report any standard strategies for identifying homologues (e.g. BLAST, PSI-BLAST, HMMer3, HHblits) together with information on the sequence repository that was searched (e.g. UniProt). 
- Report the methods used for aligning the homologues and any measures of alignment quality e.g. information content. 
- Report any key features of the method (e.g. residue potentials, co-variation, energy minimisation strategies and other optimization tools)
- Report any quality measurements (publicly available methods include NDOPE, QMEANDisCo, ModFOLD, ProQ3, VoroMQA; recent methods for model quality estimation are continuously benchmarked at https://cameo3d.org/qe)

## Section Protein Functional Site Prediction

…
Protein Complex Prediction - to be added, 3D-BioInfo Activity link
Protein Ligand Interaction Prediction - to be added 3D-Bio link
Protein Nucleic Acid Interaction Prediction - to be added 3D-Bio link

## Relevant tools and resources  

Public repositories for depositing predicted 3D-models
ModelArchive - https://www.modelarchive.org/ 
Genome3D - http://www.genome3d.net/ 

Public repositories of experimental structural data: 
Protein Data Bank (PDB) - http://www.wwpdb.org/ 
Protein Data Bank in Europe (PDBe) - https://www.ebi.ac.uk/pdbe/ 
Electron Microscopy Data Bank (EMDB) - https://www.ebi.ac.uk/pdbe/emdb/ 

Public repositories of sequence data and protein family alignments
UniProt - https://www.uniprot.org/ 
InterPro - https://www.ebi.ac.uk/interpro/  provides links to protein family resources providing alignments

Validation of model quality 
QMEANDisCo - https://swissmodel.expasy.org/qmean/  
ModFOLD - http://www.reading.ac.uk/bioinf/ModFOLD/ 
ProQ3 - https://proq3.bioinfo.se/ 
VoroMQA - http://bioinformatics.ibt.lt/wtsam/voromqa 
nDOPE - https://onlinelibrary.wiley.com/doi/full/10.1110/ps.062416606 

Assessment of protein structure prediction methods
CAMEO - https://www.cameo3d.org/  
CASP - https://predictioncenter.org/ 




{% include toollist.html tag="struct bioinfo” %}

